### PR TITLE
new bj_oji_cree folder for repository

### PR DESCRIPTION
**Pull request by billjancewicz**
I have updated the keyboard coding slightly to comply with the newer system stores that replace the header statements used in earlier versions of Keyman. I also did a minor debugging of the .kmn file at lines 141-142, code lines that were (apparently) redundant, because the compiled keyboard works fine without them. Now you can "pull a few levers and turn some knobs to get the keyboard online". This Oji-Cree one is the highest priority at this time, but there will be a couple others coming, a few more varieties of Cree (Western, Woods, Eastern, Plains). The "bj_oji_cree.php" help file will need some massaging from you, but I did use the "N'ko" help file as a model, as best I could do being a novice at editing php/html files. I left a copy of the help file in the "extras" folder in the repository. Thanks for your patient coaching, and putting up with my kludgy ways!

### CODE REVIEW

File: bj_oji_cree/HISTORY.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Template Change History
+=======================
+
+1.0 (1 Jan 2015)
```
Comment (mcdurdin) [inline bj_oji_cree/HISTORY.md]: Please add some version history here - even if just an initial date and commit version.

File: bj_oji_cree/HISTORY.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Template Change History
```
Comment (mcdurdin) [inline bj_oji_cree/HISTORY.md]: This should titled bj_oji_cree Change History

File: bj_oji_cree/HISTORY.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+=======================
+
+1.0 (1 Jan 2015)
+-----------------
+
+* Created by you
```
Comment (mcdurdin) [inline bj_oji_cree/HISTORY.md]: Change this initial history entry to something that makes sense for your keyboard

File: bj_oji_cree/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+Copyright (C) 2016 Bill Jancewicz
+
+Version 9.0
+
+Oji-Cree Phonetic-style keyboard for Unicode Canadian Aboriginal Syllabics
+
+Links
+-----
+
+ * Home:     <___URL___>
```
Comment (mcdurdin) [inline bj_oji_cree/README.md]: If you don't have a home page, use http://keyman.com/ or http://help.keyman.com/keyboard/bj_oji_cree

File: bj_oji_cree/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+ * Deployed: <http://s.keyman.com/keyboard/bj_oji_cree>
+ * Help:     <http://help.keyman.com/keyboard/bj_oji_cree>
+ * Contact:  <___EMAIL___>
```
Comment (mcdurdin) [inline bj_oji_cree/README.md]: If you don't want to provide your own email, use support@keyman.com

**Comment by billjancewicz** [comment]
Thanks for the pointers on writing the README.md and HISTORY.md files.

**Pull request closed without merging**